make error

sudo make
make  all-recursive
make[1]: Entering directory `/home/user/Downloads/ios-webkit-debug-proxy-master'
Making all in src
make[2]: Entering directory `/home/user/Downloads/ios-webkit-debug-proxy-master/src'
/bin/bash ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I../include   -I/usr/local/include -I/usr/include/libxml2   -I/usr/local/include -I/usr/include/libxml2   -g -O2 -Wall -Werror -MT websocket.lo -MD -MP -MF .deps/websocket.Tpo -c -o websocket.lo websocket.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../include -I/usr/local/include -I/usr/include/libxml2 -I/usr/local/include -I/usr/include/libxml2 -g -O2 -Wall -Werror -MT websocket.lo -MD -MP -MF .deps/websocket.Tpo -c websocket.c  -fPIC -DPIC -o .libs/websocket.o
websocket.c: In function 'ws_send_frame':
websocket.c:318:9: error: implicit declaration of function 'htons' [-Werror=implicit-function-declaration]
         uint16_t sz16 = htons(payload_length);
         ^
websocket.c:327:7: error: value computed is not used [-Werror=unused-value]
       *out_tail++;
       ^
cc1: all warnings being treated as errors
make[2]: *** [websocket.lo] Error 1
make[2]: Leaving directory `/home/user/Downloads/ios-webkit-debug-proxy-master/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/user/Downloads/ios-webkit-debug-proxy-master'
make: *** [all] Error 2

--- a/src/websocket.c
+++ b/src/websocket.c
@@ -25,6 +25,10 @@
 #endif
 #endif
 
+#ifndef htons
+#include <arpa/inet.h>
+#endif
+
 typedef int8_t ws_state;
 #define STATE_ERROR 1
 #define STATE_READ_HTTP_REQUEST 2
@@ -322,9 +326,7 @@ ws_status ws_send_frame(ws_t self,
         memcpy(out_tail, &sz64, payload_n);
     }
 
-    int i;
-    for (i = 0; i < payload_n; i++) 
-      *out_tail++;
+    out_tail += payload_n;
   }
 
   if (is_masking) {

